qwerty_ast/src/ast.rs

```diff
@@ -500,7 +500,7 @@ where
     }
 }
 
-impl<E: ToPythonCode> ToPythonCode for FunctionDef<E> {
+impl<E: ToPythonCode + fmt::Debug> ToPythonCode for FunctionDef<E> {
```

Comment:
```suggestion
impl<E: ToPythonCode> ToPythonCode for FunctionDef<E> {
```

I tried building without this and it worked